library/Fission/User/Registration/Types.hs

```diff
@@ -26,7 +29,8 @@ instance ToJSON Registration where
 
 instance FromJSON Registration where
   parseJSON = withObject "Registration" \obj -> do
-    username <- obj .:  "username"
+    username <- obj .: "username"
+    password <- obj .: "password"
```

Comment:
Does this need to be `.:?` since password isn't always included?

Reply:
Good question! It's only `.:?` if you want to provide a default value. We're saying here that it could be `Nothing`, and that's a valid value!

Reply:
Oh cool cool :+1:

Reply:
(Context: this is for a JSON parser, which may have a different representation than the underlying data structure)